src/azure-cli/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -24,6 +24,66 @@
+                                       metadata=metadata)
+        return client.create(resource_group_name=resource_group_name, account_name=account_name,
+                             container_name=container_name, blob_container=blob_container)
+    return client.create(resource_group_name=resource_group_name, account_name=account_name,
+                         container_name=container_name, public_access=public_access, metadata=metadata)
+
+
+def update_container_rm(cmd, client, container_name, resource_group_name, account_name,
+                        metadata=None, public_access=None,
+                        default_encryption_scope=None, deny_encryption_scope_override=None):
+    container = client.get(resource_group_name=resource_group_name,
+                           account_name=account_name, container_name=container_name)
+    if cmd.supported_api_version(min_api='2019-06-01', resource_type=ResourceType.MGMT_STORAGE):
+        BlobContainer = cmd.get_models('BlobContainer', resource_type=ResourceType.MGMT_STORAGE)
+        blob_container = BlobContainer(
+            metadata=metadata if metadata is not None else container.metadata,
+            public_access=public_access if public_access is not None else container.public_access,
+            default_encryption_scope=default_encryption_scope
+            if default_encryption_scope is not None else container.default_encryption_scope,
+            deny_encryption_scope_override=deny_encryption_scope_override
+            if deny_encryption_scope_override is not None else container.deny_encryption_scope_override,
+        )
+        return client.update(resource_group_name=resource_group_name, account_name=account_name,
+                             container_name=container_name, blob_container=blob_container)
+    return client.update(resource_group_name=resource_group_name,
+                         account_name=account_name, container_name=container_name,
+                         metadata=metadata if metadata is not None else container.metadata,
+                         public_access=public_access if public_access is not None else container.public_access)
+
+
+def list_container_rm(cmd, client, resource_group_name, account_name, include_deleted=None):
+    ListContainersInclude = cmd.get_models('ListContainersInclude', resource_type=ResourceType.MGMT_STORAGE)
+    include = ListContainersInclude("deleted") if include_deleted is not None else None
+
+    return client.list(resource_group_name=resource_group_name, account_name=account_name, include=include)
+
+
+def container_rm_exists(client, resource_group_name, account_name, container_name):
+    from msrestazure.azure_exceptions import CloudError
+    try:
+        container = client.get(resource_group_name=resource_group_name,
+                               account_name=account_name, container_name=container_name)
+        return container is not None
+    except CloudError:
+        return False
```

Comment:
Two suggestions here:
1. CloudError is not recommended to use here because I am afraid that there is exception type change between track1 and track2 SDK. In this way, when track2 SDK is applied, your check will be unused but introduce dependency on msrestazure.
2. When there is  error occurred, if it is not missing or not found error, we should raise the exception. please see SDK implementation.
```
        except AzureHttpError as ex:
            _dont_fail_not_exist(ex)
            return False
```